README.md

```diff
@@ -195,274 +227,424 @@ print("t-critical value:")                  # Check the t-critical value
+margin_of_error = t_critical * se
+
+# Calculate the confidence interval using margin_of_error
+confidence_interval = (sample_mean - margin_of_error,
+                       sample_mean + margin_of_error)
+
+# Check the confidence interval
+print("Confidence Interval:")
+print(confidence_interval)
+```
+
+    Confidence Interval:
+    (18.431843086289952, 21.309734172653762)
+
+
+### Verifying Our Results with `stats.t.interval`
+
+The previous exercise walked through each of the parts of calculating the confidence interval in order to reinforce your understanding of the formula. However, as you may recall from the previous lesson, there is a function within SciPy that will perform a lot of the same logic with a single function call!
+
+Let's verify our calculations by using the Python function `stats.t.interval()`:
 
 
 ```python
-stats.t.interval(alpha = 0.95,              # Confidence level
-                 df= 24,                    # Degrees of freedom
+# Run this cell without changes
+
+ci = stats.t.interval(alpha = 0.95,         # Confidence level
+                 df = sample_size - 1,      # Degrees of freedom
                  loc = sample_mean,         # Sample mean
-                 scale = sigma)             # Standard deviation estimate
-# (18.4609156900928, 21.280661568850913)
+                 scale = se)                # Standard error
+
+print("True Population Mean:")
+print(population_mean)
+print("95% Confidence Interval of Mean Based on Sample:")
+print(ci)
+
+# True Population Mean:
+# 21.00857750766395
+# 95% Confidence Interval of Mean Based on Sample:
+# (18.431843086289952, 21.309734172653762)
 ```
 
 
 ```python
 # __SOLUTION__ 
-stats.t.interval(alpha = 0.95,              # Confidence level
-                 df= 24,                    # Degrees of freedom
+
+ci = stats.t.interval(alpha = 0.95,         # Confidence level
+                 df = sample_size - 1,      # Degrees of freedom
                  loc = sample_mean,         # Sample mean
-                 scale = sigma)             # Standard deviation estimate
-```
+                 scale = se)                # Standard error
 
+print("True Population Mean:")
+print(population_mean)
+print("95% Confidence Interval of Mean Based on Sample:")
+print(ci)
+```
 
+    True Population Mean:
+    21.00857750766395
+    95% Confidence Interval of Mean Based on Sample:
+    (18.431843086289952, 21.309734172653762)
 
 
-    (18.4609156900928, 21.280661568850913)
+Going forward, if you know that the confidence interval is the desired end result, you can just use the above function instead of calculating the critical t and margin of error "by hand".
 
+### Interpreting the Confidence Interval
 
+We can see that the calculated confidence interval (`ci`) includes the true population mean (`population_mean`) calculated above. That makes sense because we chose a 95% confidence level, meaning we're 95% confident that the computed confidence interval contains the true mean.
```

Comment:
'''
That makes sense because we chose a 95% confidence level, meaning we're 95% confident that the computed confidence interval contains the true mean.
'''

This is incorrect, and is an interpretation that was specifically rebuffed in one of the original papers introducing confidence intervals, but it's an *extremely common* misinterpretation and is unfortunately all over the internet, including in some college material ([eg here])(https://sphweb.bumc.bu.edu/otlt/MPH-Modules/BS/BS704_Confidence_Intervals/BS704_Confidence_Intervals_print.html) that should be worded more precisely. 

Good ol' wikipedia has [an excellent summary](https://en.wikipedia.org/wiki/Confidence_interval#Meaning_and_interpretation) of the issues involved. 

Essentially, I think the best way of thinking through the interpretation of a confidence interval is to start from what we have `confidence` in. 

We don't have `confidence` that any given confidence interval will contain the population parameter; we have `confidence` that our **confidence-interval generating procedure** will generate confidence intervals at such a rate that, after repeatedly sampling and generating CIs from those samples, a given %-age of them will contain the true population parameter.

Note that this is exactly what the final visualization in this lab shows; after generating 20 confidence intervals from 20 samples at alpha=.05, only 5% of them (ie, 1) doesn't contain the true population parameter. 

So, with the material gone through in this lab, I think it makes sense to replace the two sentences above with the following:

'''
We can see that the calculated confidence interval (`ci`) includes the true population mean (`population_mean`) calculated above.  

However, it is a mistake to think: "Since we calculated a confidence interval at a level of 95%, we are 95% confident that the interval we calculated contains the true population mean"

Instead, we should think: "Since we generated a 95% confidence interval, we would expect that, if we repeatedly drew samples and calculated confidence intervals in the same way,  the true population mean would show up in 95% of the confidence intervals we would calculate."

We will illustrate (and visualize) this point below!
'''